src/launcher/src/test/java/engineering/everest/lhotse/functionaltests/scenarios/ApplicationFunctionalTests.java

```diff
@@ -100,4 +110,56 @@ void newUsersCanRegisterTheirOrganizationAndCreateNewUsersInTheirOrganization()
         var newUserRequest = new NewUserRequest("bob@example.com", "bobalsorocks", "My name is Bob");
         apiRestTestClient.createUser(newOrganizationId, newUserRequest, CREATED);
     }
+
+    @Data
+    @NoArgsConstructor
+    @AllArgsConstructor
+    static class ErrorResponse {
+        private HttpStatus status;
+        private String message;
+        private Map<String, String> timeStamp;
+    }
+
+    @Test
+    void jsr303errorMessagesAreInternationalized() throws JsonProcessingException {
+        apiRestTestClient.createAdminUserAndLogin();
+
+        var newOrganizationRequest = new NewOrganizationRequest("ACME", "123 King St", "Melbourne",
+                "Vic", "Oz", "3000", null, null, null, "admin@example.com");
+        var newUserRequest = new NewUserRequest("a-user", "password", "");
+        var organizationId = apiRestTestClient.createRegisteredOrganization(newOrganizationRequest, CREATED);
+
+        var response = webTestClient.post().uri("/api/organizations/{organizationId}/users", organizationId)
+                .header("Authorization", "Bearer " + apiRestTestClient.getAccessToken())
+                .header("Accept-Language", "de-DE")
+                .contentType(APPLICATION_JSON)
+                .body(fromValue(newUserRequest))
+                .exchange()
+                .returnResult(ErrorResponse.class)
+                .getResponseBody()
+                .blockFirst();
+        assertEquals("displayName: darf nicht leer sein", response.getMessage());
```

Comment:
I am not quite sure how this works because this is not part of the message files supplied by us. I assume this translation is provided out of the box by Spring?

Reply:
That's right. It's the built-in validation magic that is JSR-303.